stdlib/public/Backtracing/Context.swift

```diff
@@ -275,8 +275,9 @@ extension arm_gprs {
   }
   #elseif arch(x86_64)
   @_silgen_name("_swift_get_cpu_context")
-  static func _swift_get_cpu_context() -> X86_64Context
+  public static func _swift_get_cpu_context() -> X86_64Context
```

Comment:
Can/should these be marked `@usableFromInline` instead of `public`?

Reply:
Seems to work, and it's clearly better, so let's do that.